scripts/bash/backendStartDev.sh

```diff
@@ -44,16 +48,16 @@ fi
 
 echo "Starting server"
 
-if [[ ${skip_internal} == true ]]; then
-    echo "Skipping internal dependency build and watch"
-    eval ${start_server}
-else
+if [[ ${inclue_internal} == true ]]; then
```

Comment:
```suggestion
if [[ ${include_internal} == true ]]; then
```

Reply:
Thanks again! I've been through the script more carefully now, and actually tested the positive case (had only tested the negative ones to make sure my friendly messages were printing before 🤦 )